jani_generator/src/jani_generator/ros_helpers/ros_timer.py

```diff
@@ -181,5 +197,45 @@ def make_global_timer_automaton(timers: List[RosTimer],
             }]
         })
         timer_automaton.add_edge(timer_edge)
-
     return timer_automaton
+
+
+def make_global_timer_scxml(timers: List[RosTimer], max_time_ns: int) -> Optional[ScxmlRoot]:
+    """
+    Create a global timer SCXML from a list of ROS timers.
+
+    :param timers: The list of ROS timers.
+    :return: The global timer SCXML.
+    """
+    """
+    Generate an SCXML model containing the timers.
+    """
+    if len(timers) == 0:
+        return None
+    global_timer_period, global_timer_period_unit = get_common_time_step(timers)
+    timers_map = {
+        timer.name: convert_time_between_units(timer.period_int, timer.unit,
+                                               global_timer_period_unit)
+        for timer in timers
+    }
+    try:
+        max_time = convert_time_between_units(
+            max_time_ns, "ns", global_timer_period_unit)
+    except AssertionError:
+        raise ValueError(
+            f"Max time {max_time_ns} cannot be converted to {global_timer_period_unit}. "
+            "The max_time must have a unit that is greater or equal to the smallest timer period.")
+    scxml_root = ScxmlRoot("global_timer_automata")
```

Comment:
```suggestion
    scxml_root = ScxmlRoot("global_timer_automaton")
```
Or are there more than one? I am not sure.